R/cost_uplift.R

```diff
@@ -0,0 +1,27 @@
+#' Uplift costs
+#'
+#' @param data episode data
+#' @param uplift
+#'
+#' @return episode data with uplifted costs
+#' @export
+cost_uplift <- function(data, uplift) {
```

Comment:
I think we need to take the data and the year here, and then work out the uplift for that year within this function (or in another function.

Reply:
Sorry, I thought of it simplistically. I dug out the uplift rate for each year and created a function to set the uplift. My way is to create a data frame with each year and each year's uplift rate, and then join the data frame with episode data. 

To my best knowledge, it should be the most efficient way to do this compared with manupilate with episode directly. But please let me know if there is a better way. Thanks!

Reply:
Cool, that looks good to me. Just having a proper check / review now, but the concept seems fine 👍